src/Nox.Types/Types/Password/PasswordTypeOptions/PasswordTypeOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.RegularExpressions;
+using System;
+
+namespace Nox.Types;
+public class PasswordTypeOptions
```

Comment:
Can we update the Schema for this new TypeOptions?
https://github.com/NoxOrg/Nox.Generator#schemas-update